docker/dev/Dockerfile-backend

```diff
@@ -0,0 +1,23 @@
+FROM python:3.6-jessie
```

Comment:
@nicusX can we name this file `Dockerfile-worker`? Technically `web` is backend too, just to avoid the confusion